README.md

```diff
@@ -100,6 +100,12 @@ try {
 - [rcs.loadMapping](docs/api/loadMapping.md)
 - [rcs.config](docs/api/config.md)
 
+## Caveats
```

Comment:
Could you add this new section on the `Content` list?

Reply:
Yes sure.